source/references/weex-variable.md

```diff
@@ -160,3 +160,36 @@ weex.supports('module/*') // null
 weex.supports('@stream/fetch') // null
 weex.supports('getComponentRect') // null
 ```
+
+## `weex.isRegisteredModule`
+
+Detect whether the specific module or method is registered.
+
+```js
+weex.weex.isRegisteredModule(moduleName: string, methodName: string): boolean
+```
+
+It can only be used to check the compatibility of the specific module or method, not support components.
```

Comment:
Suggestion: It can only be used to check compatibility of a specific module or method. Checking compatibility of components is not supported.

Reply:
OK, I'll modify it.

Reply:
Let's change it to only what is supported - 
It can only be used to check compatibility of a specific module or method.